netlify/functions/manage-findings.ts

```diff
@@ -63,27 +75,22 @@ async function getFinding(
     headers: {
       "Content-Type": "application/json",
     },
-    body: JSON.stringify({error: "problem fetching finding"}),
+    body: JSON.stringify({ error: "problem fetching finding" }),
   };
 }
 
 async function getFindings(
   username: string,
-  contestId: number,
-  includeTeams: boolean = true
+  includeTeams: boolean = true,
+  contest: Contest
 ): Promise<Response> {
   // first phase:
   // given active! contest id
   // [x] warden can see own findings
   // [x] warden can see team findings
-  // [ ] can see specific finding
-  // [ ] team findings optional? (query param)
-
-  const contest = await getContest(contestId);
-
-  if (!isContestActive(contest)) {
-    // throw?
-  }
+  // [x] can see specific finding
+  // [x] team findings
+  // [ ] make team findings optional? (query param)
```

Comment:
?

```suggestion
  // [x] make team findings optional? (query param)
```

Reply:
Oh, is this actually read from the query param? I thought that part was commented out.

Reply:
I guess in the context of this function, it is optional. The caller just doesn't actually pull the value from query params. I'll add a todo there instead.